project/api/management/commands/fetch_treaties.py

```diff
@@ -0,0 +1,76 @@
+"""
+Chron.
+Copyright (C) 2018 Alisa Belyaeva, Ata Ali Kilicli, Amaury Martiny,
+Daniil Mordasov, Liam O'Flynn, Mikhail Orlov.
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <https://www.gnu.org/licenses/>.
+"""
+
+from django.core.management.base import BaseCommand, CommandError
+from django.contrib.gis.geos import Point
+from api.models import CachedData
+from datetime import datetime
+from jdcal import gcal2jd
+from math import ceil
+import re
+import os
+import requests
+
+class Command(BaseCommand):
+    help = 'Populate cached datas with treaties.'
+
+
+    def handle(self, *args, **options):
+        URL = "https://query.wikidata.org/sparql"
+        QUERY = """
+        SELECT ?treaty ?treatyLabel ?time ?location ?coors
+        WHERE
+        {
+        ?treaty wdt:P31 wd:Q625298.
+        ?treaty wdt:P585 ?time.
+        OPTIONAL {
+            ?treaty wdt:P276 ?location.
+            ?location wdt:P625 ?coors
+        }
+        FILTER (?time > "1700-01-01T00:00:00Z"^^xsd:dateTime)
+        FILTER (?time < "2019-01-01T00:00:00Z"^^xsd:dateTime)
```

Comment:
I guess this one also can be removed

Reply:
We're storing pre-1700 treaties now? Or just the 2019 filter

Reply:
Sorry just read your above message, will change to 1600

Reply:
I think he meant 2019, no? @MiklerGM

Reply:
I was thinking about those proposals
- Remove filter ```FILTER (?time < "2019-01-01T00:00:00Z"^^xsd:dateTime)```
- Extract the earliest date as a variable to share it between commands
- Extend period from 1700 to 1600